modules/mobile/docs/openapi.yaml

```diff
@@ -4833,3 +4871,12 @@ components:
       name: Authorization
       in: header
       description: Authentication is via an access token returned from IAM's SSOe service. Pass the access token in the 'Authorization' Header in the format 'Bearer {accessToken}'. e.g. 'Bearer abcdefg1234567'.
+  parameters:
```

Comment:
It looks like this was an oversight from Jason's recent work. Looking at our [api docs](https://department-of-veterans-affairs.github.io/va-mobile-app/api/#operation/get_folder_threads), you can see that Jason's entry for the route `/v0/messaging/health/folders/{id}/threads` doesn't show this section. So this will also fix that. 

We should switch all of our docs over to use this component. We have a ticket open for cleaning up the openapi docs and this would be a nice thing to do as part of that ticket. 

@Tonksthebear While looking at this, I also noticed you left commas after the param names like `pageSize,`. Can we remove those? Might be easiest just to do it as part of this PR.

Reply:
Ya the commas definitely weren't intentional